webapp/templates/webapp/project/detail.html

```diff
@@ -0,0 +1,21 @@
+{% extends '../__base.html' %}
+{% load i18n %}
+
+{% block body %}
+<h3>{{ project.project_name }}</h3>
+<a href="{% url 'project-edit' project.id %}" class="btn btn-secondary">{% trans 'Editar' %}</a>
+<a href="{% url 'project-list' %}" class="btn btn-link">{% trans 'Volver' %}</a>
++<br>
```

Comment:
La relación entre proyecto y grupo aún no está definida o sea que aún no se puede hacer este `for`.